packages/ui/src/hooks/useLedger.ts

```diff
@@ -74,6 +74,10 @@ export function useLedger (genesis?: string | null, accountIndex = 0, addressOff
     } else if (error?.includes('AbortError: The transfer was cancelled') ||
     error?.includes('No device selected') ||
     error?.includes("Failed to execute 'requestDevice' on 'USB': Must be handling a user gesture to show a permission request") ||
+    // Strangely enough this error is thrown even while importing an account.
+    // @FIXME distinguish it from actual tx rejection, once we're able to sign with ledger.
+    error?.includes('Error: Transaction rejected') ||
```

Comment:
Yup.

From when I noticed, this error showed up after I had the Polymesh app open, and then the Ledger locked by itself from idling. After unlocking the Ledger, the Polymesh app was still open, and everything was working as expected.

Need to confirm by reproducing consistently though.